### PR TITLE
Fixed function ordering issue in process_data

### PR DESCRIPTION
**Pull request by drew-sinha**
Moved the functions up like we discussed.

### CODE REVIEW

File: elegant/worm_data.py [hunk 1]
```diff
@@ -209,6 +209,51 @@ def meta_worms(grouped_worms, *features, age_feature='age', summary_features=('l
         meta_worms.append(meta_worm)
     return meta_worms
 
+def _valid_values(array):
```
Comment (zpincus) [inline elegant/worm_data.py]: I think `_valid_values` can probably stay down below out of sight, no? :)

**Comment by drew-sinha** [comment]
Resolved

**Comment by drew-sinha** [comment]
Added this second bugfix for another issue in scan_experiment_dir (the old del statement had position_name as a string).

**Comment by zpincus** [comment]
> the old del statement had position_name as a string

yep, I've done that too. Good catch.

**Pull request merged**